src/frontend/screens/MapScreen/index.tsx

```diff
@@ -52,6 +52,10 @@ export const MapScreen = () => {
 
   const styleUrlQuery = useMapStyleJsonUrl();
 
+  const styleUrl = process.env.EXPO_PUBLIC_E2E_TEST
+    ? 'mapbox://styles/mapbox/streets-v11'
+    : (styleUrlQuery?.data ?? 'mapbox://styles/mapbox/streets-v11');
```

Comment:
not sure if having the fallback here is desirable, as it could mean a rather drastic change in the map style once the query does load e.g. if the app is using a custom map, the user would see the mapbox fallback you've specified first and then switch to the custom map (which is very likely to be visually different)

```suggestion
    : styleUrlQuery.data;
```

Reply:
OK took out fallback (but irrelevant now that I moved to the hook).